docs/releasing/README.md

```diff
@@ -0,0 +1,80 @@
+# Releasing
+
+## When to release
+
+- When Coreth needs to release a new feature or bug fix.
+
+## Procedure
+
+### Release candidate
+
+ℹ️ you should always create a release candidate first, and only if everything is fine, you can create a release.
+
+In this section, we create a release candidate `v0.15.0-rc.0`. We therefore assign these environment variables to simplify copying instructions:
+
+```bash
+export VERSION_RC=v0.15.0-rc.0
+export VERSION=v0.15.0
+```
+
+1. Create your branch, usually from the tip of the `master` branch:
+
+    ```bash
+    git fetch origin master:master
+    git checkout master
+    git checkout -b "releases/$VERSION_RC"
+    ```
+
+1. Modify the [plugin/evm/version.go](../../plugin/evm/version.go) `Version` global string variable and set it to the desired `$VERSION`.
+1. Ensure the AvalancheGo version used in [go.mod](../../go.mod) is [its last release](https://github.com/ava-labs/avalanchego/releases). If not, upgrade it with, for example:
+
+    ```bash
+    go get github.com/ava-labs/avalanchego@v1.13.0
+    go mod tidy
+    ```
+
+    And fix any errors that may arise from the upgrade. If it requires significant changes, you may want to create a separate PR for the upgrade and wait for it to be merged before continuing with this procedure.
```

Comment:
I suppose it's always recommended to use the latest avalanchego version to avoid any cycle dependency problems right?